packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -287,4 +288,49 @@ describeSpec('Limbo Documents:', [], () => {
         .expectEvents(query, { removed: [doc2] });
     }
   );
+
+  specTest(
+    'Limbo documents are resolved after primary tab failover',
+    ['multi-client'],
+    () => {
+      const query = Query.atPath(path('collection'));
+      const doc1 = doc('collection/a', 1000, { key: 'a' });
+      const doc2 = doc('collection/b', 1001, { key: 'b' });
+      const deletedDoc2 = deletedDoc('collection/b', 1005);
+
+      return (
+        client(0, false)
```

Comment:
FWIW I couldn't remember or guess what this boolean argument was.  Please add `/*gcEnabled=*/` or make it so we can do .client(0).withGcEnabled(false)... or something.

Reply:
Added the "gcEnabled" comment.

I am not able to use `withGcEnabled` since `client(0)`  already drains the queue, and `withGcEnabled` rightfully validates that no steps have been added.